### ISSUE TITLE
I cand integrate it in latest version of hass.io (0.107.7)

### ISSUE THREAD

**Issue by ibBogdan** [hacs/integration#1025]
Updated hass.io to latest 0.107.7 and could not get HACS to integrate. 
Downgraded to 0.107.6 and worked just fine

**Comment by agrimpelhuber** [comment]
Since @ibBogdan isn't willing or able to give more specific input, I'm trying to fill in.

Steps to reproduce AND current versions

- Running current HASS.IO and home assistant
- Previously ran YAML integrated HACS without problems. After 0.107.7 update, Integration link vanished from sidebar menu.
- Taking up the hint that YAML integration is deprecated, deinstalled HACS completely (yes, lots of restarts and cache cleanups), and reinstalled according to the official HACS integration documentation.
- Still no link to HACS in the sidebar menu

Set HACS logging to debug. No direct output *at all*, but rather some HACS-related output from the config section:

```
2020-03-29 23:01:56 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry  for hacs
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 216, in async_setup
    hass, self
  File "/config/custom_components/hacs/__init__.py", line 81, in async_setup_entry
    startup_result = await hacs_startup()
  File "/config/custom_components/hacs/__init__.py", line 121, in hacs_startup
    lovelace_info = await system_health_info(hacs.hass)
  File "/usr/src/homeassistant/homeassistant/components/lovelace/__init__.py", line 237, in system_health_info
    return await hass.data[DOMAIN]["dashboards"][None].async_get_info()
  File "/usr/src/homeassistant/homeassistant/components/lovelace/dashboard.py", line 111, in async_get_info
    return _config_info(self.mode, self._data["config"])
  File "/usr/src/homeassistant/homeassistant/components/lovelace/dashboard.py", line 216, in _config_info
    "resources": len(config.get("resources", [])),
TypeError: object of type 'NoneType' has no len()
```

Looking up the version data from my system gave me a clue, but I can't make anything out of it. 

![image](https://user-images.githubusercontent.com/35150551/77861310-62c1c580-7214-11ea-9216-efaa327a2e56.png)

The "object of type 'NoneType' has no len()" appears where some Lovelace UI information should be (? can't verify ?). So is the HACS setup stumbling on some misconfiguration / error in System Health?

**Comment by ludeeus** [comment]
That is correct @agrimpelhuber it has been fixed in HA, but not sure when it will be released.

**Comment by agrimpelhuber** [comment]
@ludeeus do you have a link to the issue / fix? Maybe I can verify that this is really the reason for the HACS failure - and the fix.

**Comment by ludeeus** [comment]
@agrimpelhuber not sure, how/where but it should be part of the current beta.

**Comment by TheLastProject** [comment]
I just upgraded my Home Assistant to 0.108.3 and HACS started appearing for me in the sidebar again and worked fine. So I guess it's fixed in 0.108.3.

**Comment by jes1417** [comment]
Neither .108, .1, .2, or .3 loaded HACS the first time but after a reboot it showed up.
